lib/docker/template/cache.rb

```diff
@@ -42,9 +42,10 @@ def aliased_context(builder)
       # --
 
       def cleanup(repo)
+        return unless repo.clean_cache?
         cache_dir = repo.cache_dir.parent
 
-        if repo.cacheable? && cache_dir.exist?
+        if cache_dir.exist?
```

Comment:
Thanks. I will have a look, and test those images from `envygeeks/docker`. As soon as I have tested them, I will report it.